### DISCUSSION TITLE
Tabs on the left instead of on top

### DISCUSSION THREAD

**Discussion by jmrichardson** [datapane/datapane#365] [Ideas]
It used to be that you could have the tabs all on the left side of the page. However, I can't figure out how to do it with the new version?  Do i have to use the old version?  For example, I have 10 tabs, but all of them are now just in a dropdown.  Thanks for any help

**Comment by mands** [comment]
Sorry for the issue here, closing and linking to #364 which covers the same issue. Essentially we're looking at ways to bring back the sidebar for top-level tabs / pages in a newer release. They were removed as part of reducing scope to release Datapane Apps.
